src/models/model_base.h

```diff
@@ -19,7 +19,7 @@ class ModelBase {
 public:
   virtual void load(Ptr<ExpressionGraph>,
                     const std::string&,
-                    bool markReloaded = true)
+                    bool markReloaded = true) const
```

Comment:
How can load() be const? I am confused; per `const`, it does not modify the Model itself. What does not modify then?

Reply:
... the graph. Take a look at src/model/encoder_decoder.cpp, for example.

Reply:
Would you mind adding a brief comment to the code?